src/udev/udev-builtin-net_id.c

```diff
@@ -897,6 +938,20 @@ static int builtin_net_id(sd_device *dev, int argc, char *argv[], bool test) {
                 return 0;
         }
 
+        /* get netdevsim path names */
+        if (names_netdevsim(dev, &names) >= 0 && names.type == NET_NETDEVSIM) {
+                char str[IFNAMSIZ];
+
+                if (names.netdevsim_path[0] &&
+                    snprintf_ok(str, sizeof str, "%s%s", prefix, names.netdevsim_path))
+                        udev_builtin_add_property(dev, test, "ID_NET_NAME_PATH", str);
+
+                if (names.netdevsim_path[0] &&
```

Comment:
Both `if`s are the same, so the same value is used for both ID_NET_NAME_PATH and ID_NET_NAME_SLOT. Is this on purpose? I don't think it makes much sense to provide the same name twice.

Reply:
Okay. I just followed what is already in the code for NET_USB and NET_BCMA. Will remove ID_NET_NAME_SLOT